openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/DataQualityTab.tsx

```diff
@@ -227,6 +235,7 @@ const DataQualityTab: React.FC<DataQualityTabProps> = ({
         }}
         pagination={false}
         rowKey="id"
+        scroll={{ x: 1550 }}
```

Comment:
Why we are adding scroll?

Reply:
@chirag-madlani  currently it is very congested in the table and even for any column if the data comes larger then it will shrink another column and will create bad UX.